releasenotes/notes/limit-trace-agent-buffers-9e7e4c9d54e7c237.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    APM: Limit the size of the buffers used to decode the request body in the trace agent. This prevents the agent from allocating memory for requests that are too large.
```

```suggestion
    APM: Limit the size of the buffers used to decode the request body in the Trace Agent. This prevents the Agent from allocating memory for requests that are too large.
```

We try to always capitalize Agent when discussing the Datadog Agent, to differentiate from other agents!